x-pack/plugins/ml/public/application/memory_usage/nodes_overview/allocated_models.tsx

```diff
@@ -122,13 +122,27 @@ export const AllocatedModels: FC<AllocatedModelsProps> = ({
       },
     },
     {
-      field: 'node.throughput_last_minute',
-      name: i18n.translate(
-        'xpack.ml.trainedModels.nodesList.modelsList.throughputLastMinuteHeader',
-        {
-          defaultMessage: 'Throughput',
-        }
+      name: (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.ml.trainedModels.nodesList.modelsList.throughputLastMinuteTooltip',
+            {
+              defaultMessage: 'The number of requests processed in the last 1 minute.',
```

Comment:
```suggestion
              defaultMessage: 'The number of documents processed per minute.',
```

Reply:
FYI I took this description from https://www.elastic.co/guide/en/elasticsearch/reference/current/get-trained-models-stats.html

Reply:
Assuming this is displaying `throughput_last_minute` then I definitely think we need to use the text `The number of requests processed in the last 1 minute.`

Reply:
Cool, thanks for clarifying!